examples/gno.land/p/demo/ownable/ownable.gno

```diff
@@ -27,7 +33,13 @@ func (o *Ownable) TransferOwnership(newOwner std.Address) error {
 		return ErrInvalidAddress
 	}
 
+	prevOwner := o.owner
 	o.owner = newOwner
+	std.Emit(
+		OwnershipTransferEvent,
+		"from", string(prevOwner),
```

Comment:
Nitpick: maybe we should use the built-in `.String()` function on addresses

Reply:
I personally don't care. Does anyone else have a strong opinion on `string(foo)` vs `foo.String()`?